app/models/piece.rb

```diff
@@ -1,4 +1,8 @@
 class Piece < ActiveRecord::Base
 	belongs_to :user
 	belongs_to :game
+
+	def self.types
+		%w(Pawn Rook Knight Bishop Queen King)
+	end
```

Comment:
`self.types` is for STI. It tells what subclasses belong to the Piece model. @mikecoutermarsh...correct me if I'm wrong here.